src/inputs/pipe.c

```diff
@@ -364,18 +406,24 @@ parse_item(struct input_metadata *m, const char *item)
     {
       pthread_mutex_lock(&pipe_metadata_lock);
 
-      if (data != &progress)
-	free(*data);
+      if (data != &progress && data != &volume)
+        free(*data);
```

Comment:
Yes, as you can see forked-daapd has a (weird) indentation style, where 8 spaces become tabs